### DISCUSSION TITLE
dashy on Proxmox LXC

### DISCUSSION THREAD

**Discussion by BeeJoo123** [Lissy93/dashy#1706] [Help and Support]
Hi,
is there any way to install dashy als lxc container in Proxmox?
I know, can be done in a small VM with Docker, but would be nil do find a LXC template ?
Ah, and my Proxmox is ion Raspberry so arm64 ?
Thanks.

**Comment by BuzzMoo** [comment]
Yes, tteck has written a handy script that will install it on a Debian LXC on Proxmox: [[](url)](https://tteck.github.io/Proxmox/#dashy-lxc)
That's what I've been using for a while now.
